run_borg_hyperv.ps1

```diff
@@ -324,18 +249,27 @@ function Main {
     Write-Host "    "
 
     Cleanup-Environment $BootResultsPath $ProgressLogsPath
-    $vmNames = Get-VMNames
-    $Backend = [HypervBackend]::new(@("localhost"))
-    Cleanup-VMS -VMNames $vmNames $VMCleanTimeout $Backend.Serialize()
+
+    $Params = @{
+        "ComputerName" = "localhost";
+        "SecretsPath" = "";
+        "BaseVHDsPath" = "$BaseVHDsPath";
```

Comment:
why do you put in quotes vars? There is no need if you do not need to concat vars